src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -133,6 +133,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
 
             $io->success(sprintf('Server listening on http://%s', $config->getAddress()));
+            if ('1' === ini_get('xdebug.profiler_enable_trigger')) {
```

Comment:
The `'1' ===` doesn't make really sense.
The really correct way to check a boolean ini setting is by using filter_var:
`filter_var(ini_get('xdebug.profiler_enable_trigger'), FILTER_VALIDATE_BOOLEAN)`
See http://php.net/manual/fr/function.filter-var.php#121263
I'd suggest to either make a loose comparison (thus my original proposal: `if (ini_get('xdebug.profiler_enable_trigger')) {`) or a strict one using filter_var, but not something in between.

Reply:
`ini_get` return the normalized value instead of the one present in the `php.ini` file, so the return value will be `'1'` ou `''`  here (or `false` if xdebug is not loaded).

Reply:
Oh great! Let's use `if (ini_get('xdebug.profiler_enable_trigger')) {` then, to be consistent with the code base at least.

Reply:
Done